api-server/openapi.yml

```diff
@@ -13,12 +13,12 @@ info:
     to get access to the [management api](https://github.com/apache/activemq-artemis/blob/main/docs/user-manual/management.adoc#the-management-api) of a broker instance via 
```

Comment:
this openapi.yml need to be sync with the one on api-server repo.
When we do code-gen we can probably use an url , something like

npx --package @7nohe/openapi-react-query-codegen@0.4.1 openapi-rq -i **https://raw.githubusercontent.com/artemiscloud/activemq-artemis-jolokia-api-server/main/openapi.yml** -c fetch -o src/openapi/jolokia && yarn pretty-quick && yarn eslint ./src/openapi --fix",
    "copy-config"

Reply:
I've thought about that to, but I think it is safer to have a hard copy within the self-provisioning-plugin. There are a couple of reasons why:

* The CI is testing that no manual changes are introduced on the generated code by running the codegen command. If the openapi definition on the `jolokia-api-server` project changes compared to the one that was used to generate the api, it would fail the ci
* There might be a time when the self-provisioning-plugin will lag behind the jolokia-api-server. For instance we could introduce a breaking change on the api-server side that takes time to integrate on the self-provisioning-plugin.

The drawback is that we need to copy the file and open a PR when the openapi definition changes.
We could, instead of copying the file use a commit hash in the url to specify the version we are compatible with.
That wouldn't change the fact that we would need to open a PR to have the self-provisioning-plugin integrate the latest changes, but that could make it more clear which version the codegen is targeting.

Reply:
ok, let keep the way it is now until a better solution is found.